kcidb/cache/__init__.py

```diff
@@ -49,6 +49,11 @@ def store(self, url):
             url:    The URL to try to cache.
         """
         object_name = self._format_object_name(url)
+
+        # Cache every 4096th URL only for the trial period
+        if not object_name.endswith("000"):
+            return
+
```

Comment:
Please use the `cache: ` tag in this commit's subject.

Reply:
okay